### ISSUE TITLE
IndexId ClassNotFoundException

### ISSUE THREAD

**Issue by Vampire** [hsz/idea-gitignore#534]
### Prerequisites

* [x] Plugin is in the latest version
* [x] Issue was not reported yet
* [x] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Description

I just updated the plugin and now my IJ log is flooded with the following exceptions

```java
java.lang.NoClassDefFoundError: com/intellij/util/indexing/IndexId
        at mobi.hsz.idea.gitignore.IgnoreManager$2.fetch(IgnoreManager.java:149)
        at com.intellij.openapi.vcs.actions.AbstractVcsAction$1.run(AbstractVcsAction.java:58)
        at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:945)
        at com.intellij.openapi.vcs.actions.AbstractVcsAction.performUpdate(AbstractVcsAction.java:55)
        at com.intellij.openapi.vcs.actions.AbstractVcsAction.performUpdate(AbstractVcsAction.java:33)
        at com.intellij.openapi.actionSystem.AsyncUpdateAction.update(AsyncUpdateAction.java:66)
        at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:120)
        at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
        at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:139)
        at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:164)
        at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:164)
        at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:96)
        at com.intellij.openapi.actionSystem.impl.ActionToolbarImpl.a(ActionToolbarImpl.java:936)
        at com.intellij.openapi.actionSystem.impl.ActionToolbarImpl.access$000(ActionToolbarImpl.java:64)
        at com.intellij.openapi.actionSystem.impl.ActionToolbarImpl$1.updateActionsImpl(ActionToolbarImpl.java:173)
        at com.intellij.openapi.actionSystem.impl.ToolbarUpdater$MyUpdateRunnable.run(ToolbarUpdater.java:191)
        at com.intellij.openapi.wm.impl.FocusManagerImpl.a(FocusManagerImpl.java:675)
        at com.intellij.openapi.wm.impl.FocusManagerImpl.j(FocusManagerImpl.java:656)
        at com.intellij.openapi.wm.impl.FocusManagerImpl.i(FocusManagerImpl.java:626)
        at com.intellij.openapi.wm.impl.FocusManagerImpl.access$200(FocusManagerImpl.java:60)
        at com.intellij.openapi.wm.impl.FocusManagerImpl$11.run(FocusManagerImpl.java:522)
        at com.intellij.util.ui.UIUtil.invokeLaterIfNeeded(UIUtil.java:2405)
        at com.intellij.openapi.wm.impl.FocusManagerImpl.doWhenFocusSettlesDown(FocusManagerImpl.java:500)
        at com.intellij.openapi.actionSystem.impl.ToolbarUpdater.a(ToolbarUpdater.java:108)
        at com.intellij.openapi.actionSystem.impl.ToolbarUpdater.access$400(ToolbarUpdater.java:40)
        at com.intellij.openapi.actionSystem.impl.ToolbarUpdater$MyTimerListener.run(ToolbarUpdater.java:163)
        at com.intellij.openapi.actionSystem.impl.WeakTimerListener.run(WeakTimerListener.java:54)
        at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.a(ActionManagerImpl.java:1431)
        at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.a(ActionManagerImpl.java:1421)
        at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.actionPerformed(ActionManagerImpl.java:1405)
        at javax.swing.Timer.fireActionPerformed(Timer.java:313)
        at javax.swing.Timer$DoPostEvent.run(Timer.java:245)
        at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
        at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
        at java.awt.EventQueue.access$500(EventQueue.java:97)
        at java.awt.EventQueue$3.run(EventQueue.java:709)
        at java.awt.EventQueue$3.run(EventQueue.java:703)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
        at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
        at com.intellij.ide.IdeEventQueue.c(IdeEventQueue.java:857)
        at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:658)
        at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:386)
        at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
        at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
        at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.intellij.util.indexing.IndexId PluginClassLoader[mobi.hsz.idea.gitignore, 2.6.1]
        at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:64)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 58 more
```

### Versions

**Plugin:** 

```
2.6.1
```

**IDE:** 

```
2016.1.4
```

**OS:** 

```
Windows 10
```

**Comment by HyfSunshine** [comment]
me,too

**Comment by treethingy** [comment]
Me, too

**Comment by prateekprshr-nith** [comment]
I am also getting the same error

**Comment by junyi5257** [comment]
same error ,
I uninstall 2.6.1  and try install  2.5.0 from disk,it is ok.

**Comment by hsz** [comment]
Thanks for this report. It's fixed and released with v2.6.2
